demos/nodejs-custom-checkpoints/README.md

```diff
@@ -0,0 +1,27 @@
+# JavaScript PowerSync + Custom Write Checkpoints
+
+This is a demo for using custom Write Checkpoints with PowerSync. Custom Write Checkpoints eliminate flicker when uploads do not directly reflect in the backend source database. This feature is only available for customers on [Team and Enterprise](https://www.powersync.com/pricing) plans.
```

Comment:
```suggestion
This is a demo for using custom Write Checkpoints with PowerSync. Custom Write Checkpoints eliminate flicker when uploads do not directly reflect in the backend source database. This feature is only available for customers on [Team and Enterprise](https://www.powersync.com/pricing) plans.

Docs are available [here](https://docs.powersync.com/usage/use-case-examples/custom-write-checkpoints).
```

Reply:
Not yet available, but that's where they should be after https://github.com/powersync-ja/mintlify-docs/pull/11/files